DEVELOPMENT.md

```diff
@@ -20,4 +30,4 @@ If you have non-trivial changes you'd like us to incorporate, please open an iss
 
 When you're ready for someone to look at your issue or PR, assign `@stytchauth/client-libraries` (GitHub should do this automatically). If we don't acknowledge it within one business day, please escalate it by tagging `@stytchauth/engineering` in a comment or letting us know in [Slack].
 
-[Slack]: https://join.slack.com/t/stytch/shared_invite/zt-nil4wo92-jApJ9Cl32cJbEd9esKkvyg
+[slack]: https://join.slack.com/t/stytch/shared_invite/zt-nil4wo92-jApJ9Cl32cJbEd9esKkvyg
```

Comment:
Ha, my autoformatter did this

Reply:
Looks like your autoformater is either too smart or not smart enough 😛 

Which one do you use? I'm curious about options for auto-formatting Markdown because I've never tried that before.